README.md

```diff
@@ -86,19 +86,6 @@ New additions will be met with skepticism. If you think there is a
 missing feature, please file a bug clearly describing the problem and 
 the alternatives you tried before submitting a PR.
 
-# Reporting security issues
```

Comment:
This section should be left in place. Reporting of security issues should not be buried in the contributing docs.